db/migrate/20150327220027_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
I do not recommend this. It is a security level threat bomb, because you expose tokens in your database and basically all your accounts can be hacked if someone steals your data.
Second reason why I do not recommend this is - you are performing data migration in schema migrations. Better way would be to create a rake task. [Check this out for creating custom rake tasks](http://guides.rubyonrails.org/command_line.html#custom-rake-tasks)